libkbfs/interfaces.go

```diff
@@ -482,10 +482,9 @@ type KeyMetadata interface {
 	GetTlfHandle() *TlfHandle
 
 	// HasKeyForUser returns whether or not the given user has
-	// keys for at least one device at the given key
-	// generation. Returns false if the TLF is public, or if the
-	// given key generation is invalid.
-	HasKeyForUser(keyGen KeyGen, user keybase1.UID) bool
+	// keys for at least one device. Returns an error if the TLF
```

Comment:
I just meant that in MDv3, we don't have key generations in the sense that we don't store them in the MD.